wp-content/themes/core/assets/css/src/theme/layouts/_variables.pcss

```diff
@@ -49,6 +49,7 @@
 	--grid-width: var(--grid-12-col);
 	--grid-width-staggered: var(--grid-10-col);
 	--grid-width-staggered-double: var(--grid-8-col);
+	--grid-width-full-bleed: 1440px;
```

Comment:
Some of our components/panels (Such as the Media & Text) have a "full bleed" option that isn't really full-bleed-viewport. Thus, this value can be used to max-width those items to a larger width that matches the outer dimensions for the designs.